chromiumoxide_pdl/src/build/event.rs

```diff
@@ -289,10 +294,10 @@ impl<'a> EventBuilder<'a> {
                                         if params.is_some() {
                                             return Err(de::Error::duplicate_field("params"));
                                         }
-                                        params = Some(match method.as_ref().ok_or_else(|| de::Error::missing_field("params"))
-                                        ?.as_str() {
+                                        let value = map.next_value::<&serde_json::value::RawValue>()?;
+                                        params = Some(match method.as_ref().ok_or_else(|| de::Error::missing_field("method"))?.as_str() {
                                             #deserialize_from_method
-                                            _=>CdpEvent::Other(map.next_value::<serde_json::Value>()?)
+                                            _ => CdpEvent::Other(serde_json::from_str::<serde_json::Value>(value.get()).unwrap())
```

Comment:
These `unwrap()` should never panic, because `RawValue` can only be deserialized from a valid JSON, and `Value` can be serialized from any valid JSON.